include/deal.II/meshworker/scratch_data.h

```diff
@@ -285,16 +285,13 @@ namespace MeshWorker
     /**
      * Same as the other constructor, using the default MappingQ1.
      *
-     * @param mapping The mapping to use in the internal FEValues objects
      * @param fe The finite element
      * @param quadrature The cell quadrature
      * @param update_flags UpdateFlags for the current cell FEValues
-     * @param neighbor_update_flags UpdateFlags for the neighbor cell FEValues
      * @param face_quadrature Face quadrature, used for FEFaceFvalues and
      * FESubfaceValues for both the current cell and the neighbor cell
      * @param face_update_flags UpdateFlags used for FEFaceFvalues and
      * FESubfaceValues for the current cell
```

Comment:
@tjhei Actually, I see that this line and the one below should be removed as well.